whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisRestService.java

```diff
@@ -127,6 +133,8 @@ public Response delete(
             ssoTranslator.populateCacheAuthToUsername(updateContext, originalObject);
             originalObject = ssoTranslator.translateFromCacheAuthToUsername(updateContext, originalObject);
 
+            addBasic(request, passwords);
```

Comment:
Can the Basic auth be treated like a password for all requests?

Reply:
yes and no.
- Yes, because we can use the passwds list that we are already using to pass the passwd to lower level. I am already using the same list
- No, because we need to check two things here, the name of the mntner TEST-MNT:X-PASWD. If we update a person objects for example it could have a list of mnts as mnt-by. We tackle this when we know that mnt-bys.

So we can not do the same in both situations. In one of them just a passwd will be checked and in the other one the passwd and the name

Reply:
Conversation over Mattermost:

Try to unify all methods to use PasswordCredentials. Get currently is just using FilterAuthFunction and not PasswordCredentials .
This will required a little refactor of the current code.

Reply:
I am creating a PasswordCredentials for the basicAuth yes. For the get response I was taking a look and it is not easy to merge everything with the update. Maybe we can merge in the future in another task, but the complexity of that task will be 5 for sure.